src/styles/mixin/card.scss

```diff
@@ -0,0 +1,117 @@
+$page-font-family: Roboto, sans-serif;
+$font-weigth-bolt: 700;
```

Comment:
There is a typo in the variable name `$font-weigth-bolt`. It should be `$font-weight-bold` to correctly reflect the intended meaning and follow standard spelling.